#ifndef ADDDIALOG_H
#define ADDDIALOG_H

#include <QDialog>
#include <qdebug.h>
#include <QSqlDatabase>
#include <QSqlError>
#include <QSqlQuery>
#include <QtGui>
#include <QSqlQueryModel>

namespace Ui {
class AddDialog;
}

class AddDialog : public QDialog
{
    Q_OBJECT

public:
    explicit AddDialog(QWidget *parent = nullptr);
    ~AddDialog();

private slots:
    void on_buttonBox_accepted();

private:
    Ui::AddDialog *ui;
};

#endif // ADDDIALOG_H
